torch/autograd/__init__.py

```diff
@@ -328,8 +329,9 @@ def variable(*args, **kwargs):
 # deprecated function - variable(...).
 variable.Variable = Variable  # type: ignore[attr-defined]
 
-if not torch._C._autograd_init():
-    raise RuntimeError("autograd initialization failed")
+if os.environ.get("PARSH_AUTORELOAD_CONTEXT") != "1":
```

Comment:
Please abstract that in a utils file so that the env variable is hard-coded in a single place.